### ISSUE TITLE
Updates to QA protocol necessary for switch to Jekyll site

### ISSUE THREAD

**Issue by khanspers** [wikipathways/academy#28]
This is a list of updates/additions to the weekly [QA protocol](https://new.wikipathways.org/academy/qaprotocol.html) that could be useful (or are necessary) when the site is switched over to the Jekyll site. 

For the transition period when both sites are live (Jekyll site is primary):
1. Idea: Add a step at the beginning of the QA track to:
  * explain that pathway edits are reviewed on the "classic" wiki site, and not on the Jekyll site (to avoid confusion)
  * ask curators to perform the [Update Approval tags](https://new.wikipathways.org/academy/stages/task-curated/index.html) multiple times during the week, mainly minimize the delay for users to see their recent edit on the new site. Also this behavior is more similar to what will be expected of the weekly curators once everything works through Github. 
  * We could even switch the order of tasks so that the Update Approval tags task comes first.

When "classic" site is retired and edits happen directly at GitHub:

2. Updates to [QA additional tasks](https://new.wikipathways.org/academy/stages/qaprotocol-extra/index.html#intro):
  * Test search: update for Jekyll site, or remove if not needed.
  * Pathways for deletion: Update to reflect how delete works on Jekyll site, or remove if not needed.
  * Instructions for passing the torch: update how to find who's next in the rotation

**Comment by khanspers** [comment]
One more task, merged from a different issue:

- Send email in response to notifications in the new-users Slack channel. These are not really new users, but rather users who's pathways/edits have been included in the Approved collection for the first time (i.e. new to GitHub). [Email template](https://docs.google.com/document/d/1WJOqBVsU9dkJb9NNsvVXusaBVTAcx3HQngzrisMv1F8/edit).

**Comment by khanspers** [comment]
Status:

- explain that pathway edits are reviewed on the "classic" wiki site, and not on the Jekyll site (to avoid confusion) => added to front page of QA protocol.

- Test search: update for Jekyll site, or remove if not needed. => we've decided to keep this as-is for now (test search on classic)
- Pathways for deletion: Update to reflect how delete works on Jekyll site, or remove if not needed. => Unclear how to handle this on new site. Leave as-is for now (meaning delete on classic).
- Instructions for passing the torch: update how to find who's next in the rotation => Skip this for now. The rotation is listed on the [index page](https://github.com/wikipathways/wikipathways.github.io/blob/main/index.md) of WP, but only usernames are listed. Google calendar sends a reminder anyway so no need to do this. Removed from the protocol.
